docs/advanced_usages/custom-tests.md

```diff
@@ -71,9 +71,6 @@ Full AntaTest API documentation is available in the [API documentation section](
 
 ### Instance Attributes
 
-!!! info
-    You can access an instance attribute in your code using the `self` reference. E.g. you can access the test input values using `self.inputs`.
-
```

Comment:
Why are we removing this? Irrelevant?

Reply:
Agreed, it should be still relevant.

Reply:
because people using ANTA as a python library should not be learning about self. We can put it back :p

Reply:
+1 @gmuloc

Reply:
Haha ok good point :100: